src/base/static/state/ducks/datasets.js

```diff
@@ -1,8 +1,43 @@
 import PropTypes from "prop-types";
 
+// Prop Types:
```

Comment:
We should start using a "prop type" section in our ducks, instead of having them interspersed throughout